src/utils/deploy.ts

```diff
@@ -137,11 +164,16 @@ export class DeployUtils {
       // Extract and return the network names
       const networkNames = networksNode.value.properties.map((network) => {
         if (network.type === 'Property' && network.key.type === 'Identifier') {
-          return network.key.name;
+          return {
+            [network.key.name]: `${network.value.properties[0].value.object.name}.${network.value.properties[0].value.property.name}`,
```

Comment:
Are we confident we will always have a value under properties[0].value.object or any fallback/null handling might be needed